tests/parse_self.rs

```diff
@@ -92,3 +92,14 @@ fn test_parse_self_debug_line() {
         }
     }
 }
+
+#[test]
+fn test_parse_self_debug_aranges() {
+    let debug_aranges = read_section("debug_aranges");
+    let debug_aranges = DebugAranges::<LittleEndian>::new(&debug_aranges);
+
+    let mut aranges = debug_aranges.aranges();
+    while let Some(_) = aranges.next_arange().expect("Should parse arange OK") {
+        // Not really anything else we can check right now.
```

Comment:
Generally, we would do `test::black_box(arange)` to make sure that the whole thing didn't get optimized away. See https://doc.rust-lang.org/book/benchmark-tests.html for more info.

I'll merge and fix this in a follow up myself, though.